storagetransfer/check_latest_transfer_operation.go

```diff
@@ -0,0 +1,54 @@
+package storagetransfer
+
+// [START storagetransfer_get_latest_transfer_operation]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	storagetransfer "cloud.google.com/go/storagetransfer/apiv1"
+	longrunningpb "google.golang.org/genproto/googleapis/longrunning"
+	storagetransferpb "google.golang.org/genproto/googleapis/storagetransfer/v1"
+)
+
+func check_latest_transfer_operation(w io.Writer, projectID string, jobName string) (*storagetransferpb.TransferOperation, error) {
+	// Your Google Cloud Project ID
+	// projectID := "my-project-id"
+
+	// The name of the job whose latest operation to check
+	// jobName := "transferJobs/1234567890"
+	ctx := context.Background()
+	client, err := storagetransfer.NewClient(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("storagetransfer.NewClient: %v", err)
+	}
+	defer client.Close()
+
+	job, err := client.GetTransferJob(ctx, &storagetransferpb.GetTransferJobRequest{
+		JobName:   jobName,
+		ProjectId: projectID,
+	})
+	if err != nil {
+		return nil, fmt.Errorf("Failed to get transfer job: %v", err)
```

Comment:
Error strings shouldn't be capitalized (here and elsewhere): https://github.com/golang/go/wiki/CodeReviewComments#error-strings